scripts/pylint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+PREFIX="$(cd "$(dirname "$0")"/.. && pwd)/prefix"
+MIN_CODE_QUALITY=8.0
+
+get_code_score() {
```

Comment:
just to make sure I'm understanding; this function doesn't get the score of some code, it sets the minimum code quality to require based on its argument, right?